packages/dapp/src/hooks/useKeysManager.ts

```diff
@@ -0,0 +1,156 @@
+import type { VerificationMethodReference } from '@windingtree/org.json-schema/types/org.json';
+import type { GenericStateRecord } from '../store';
+import { useState, useCallback } from 'react';
+import { useAppReducer } from '../store/reducer';
+
+export const allowedKeysTypes: string[] = ['EcdsaSecp256k1VerificationKey2019'];
+
+export type KeyType =
+	typeof allowedKeysTypes[number];
+
+export enum RevocationReason {
```

Comment:
It is possible to extract RevocationReason and values from the ORG.JSON schema
```
import type { VerificationMethodReference } from '@windingtree/org.json-schema/types/org.json';
import { object } from '@windingtree/org.id-utils';
import { org } from '@windingtree/org.json-schema;

export const type RevocationReference = VerificationMethodReference['verificationMethodRevocation'].reason;
export const revocationReasonValues = object.getDeepValue(org. 'definitions.VerificationMethodRevocationReference.properties.reason.enum');
```

Reply:
✔️